src/utils/icons/style.ts

```diff
@@ -1,20 +0,0 @@
-import styled from '@emotion/styled';
-
-export const Icons = styled.div`
```

Comment:
These were only ever used for Storybook anyway, and so were already not used since I did not catch that in my earlier Storybook conversion work.